libs/component-library/src/lib/tag/tag.tsx

```diff
@@ -26,6 +26,7 @@ const FxTag = (props: FxTagProps) => {
       height={26}
       justifyContent="center"
       paddingHorizontal="8"
+      alignSelf="flex-start"
```

Comment:
This might be confusing as the default behaviour as parent elements would apear to not control the alignment.